terraform/devnet-0/digitalocean.tf

```diff
@@ -227,6 +227,45 @@ resource "digitalocean_firewall" "main" {
   depends_on = [digitalocean_project_resources.droplets]
 }
 
+resource "digitalocean_firewall" "bootnode" {
```

Comment:
Do firewall rules get applied in addition? Or is it one set for all? 

If so, we might need to allow all the p2p ports on this one too? OR if the ethereum node firewall rules apply then we don't need to redefine outbound rules?

Reply:
"must be" - I yolo added it lmfao. Didn't verify if we had inbound connections working on the relay actually.

Reply:
fixed with a additional tag (bootnode:<network-name>) that is only set for bootnodes